### DISCUSSION TITLE
GitHub Copilot could not connect to server in Setp 1 Process

### DISCUSSION THREAD

**Discussion by gowthami2019** [skills/.github#323] [Code with Copilot]
GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as gowthami2019."I had enable Gitub copliot in perference->settings->enable Github copliot.

My error:
![Capture](https://github.com/skills/.github/assets/48951060/dffb3011-cfff-4682-b49d-83ed53720581)

**Comment by cmwilson21** [comment]
👋 @gowthami2019 - If you are still having issues, this would best be handled by our [support team](https://support.github.com/request). Please reach out to them. 💛
